ReminderApp-OBJC/ReminderApp-OBJC/Classes/Model/CDUser+CoreDataProperties.h

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
Remove this class form the model. We will use the Keychain and it is not needed in Core Data anymore.